packages/rrweb-player/src/Controller.svelte

```diff
@@ -438,6 +448,7 @@
   <div class="rr-controller">
     <div class="rr-timeline">
       <span class="rr-timeline__time">{formatTime(currentTime)}</span>
+      <!-- svelte-ignore a11y-no-static-element-interactions -->
```

Comment:
?

Reply:
just for the IDE, otherwise it highlights the whole block